### PR TITLE
cmd/ethkey: add command to change a keyfile passphrase

### PR DESCRIPTION
**Pull request by stevenroose**
Introduces `ethkey changepassphrase <keyfile>` to change the passphrase from a keyfile.

### CODE REVIEW

File: cmd/ethkey/main.go [hunk 1]
```diff
@@ -46,7 +47,7 @@ func init() {
 // Commonly used command line flags.
 var (
 	passphraseFlag = cli.StringFlag{
-		Name:  "passwordfile",
+		Name:  "passfile",
```
Comment (fjl) [inline cmd/ethkey/main.go]: Please don't change the name of this option.

File: cmd/ethkey/utils.go [hunk 2]
```diff
@@ -57,9 +45,29 @@ func getPassPhrase(ctx *cli.Context, confirmation bool) string {
...
 		}
 	}
+
 	return passphrase
 }
 
+// getPassPhrase obtains a passphrase given by the user.  It first checks the
+// --passfile command line flag and ultimately prompts the user for a
+// passphrase.
+func getPassphrase(ctx *cli.Context) string {
```
Comment (fjl) [inline cmd/ethkey/utils.go]: Please fix the function name in the documentation comment.

File: cmd/ethkey/changepassphrase.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+var commandChangePassphrase = cli.Command{
+	Name:      "changepassphrase",
+	Usage:     "change the passphrase on a keyfile",
+	ArgsUsage: "<keyfile>",
+	Description: `
+Change the passphrase of a keyfile.`,
+	Flags: []cli.Flag{
+		passphraseFlag,
+		cli.StringFlag{
+			Name:  "newpassfile",
```
Comment (fjl) [inline cmd/ethkey/changepassphrase.go]: I see that you renamed the existing flag to match this one. `--newpasswordfile` sounds bad, but it's probably the better option so we don't break existing scripts.

**Comment by stevenroose** [comment]
Yeah I realized I fucked up with the password vs passphrase before :/

Do you prefer me to use password everywhere then?

**Comment by fjl** [comment]
Yes it should be `password` because that's what we have now.

**Comment by stevenroose** [comment]
Hmm, yeah I know it is bad to make breaking changes. However, since the ethkey tool is this new, I think it might make sense to reconsider. The entire geth system uses the word passphrase. This is one single mistake that breaks this rule. I really think it makes sense to make this breaking change in a niche feature of a niche cli tool instead of having the entire tool speak a different word :)

It's your call, though.

**Comment by stevenroose** [comment]
You misspelled password in the first change of your commit.

**Comment by fjl** [comment]
Thank you

**Pull request merged**